test/unit/io/alignment_file/format_sam_test.cpp

```diff
@@ -19,7 +19,7 @@ struct alignment_file_read<format_sam> : public alignment_file_data
 	using stream_type = std::istringstream;
 
     std::string big_header_input{
-R"(@HD	VN:1.0	SO:coordinate	SS:coordinate:queryname	GO:none
+R"(@HD	VN:1.6	SO:coordinate	SS:coordinate:queryname	GO:none
 @PG	ID:qc	PN:quality_control	CL:qc -f file1	DS:trim reads with low qual	VN:1.0.0
 @PG	ID:novoalign	PN:novoalign	VN:V3.02.07	CL:novoalign -d /path/hs37d5.ndx -f /path/file.fastq.gz	PP:qc
```

Comment:
huh? This is just an example SAM file. The PG tag describes the program/command pipeline of "how this SAM file was created".. I copy and pasted this from the SAM specs. Of course I could write an example with yara here but since it is just for testing I don't see the issue?